publication-commons/src/main/java/no/unit/nva/publication/model/business/publicationstate/UpdatedResourceEvent.java

```diff
@@ -0,0 +1,30 @@
+package no.unit.nva.publication.model.business.publicationstate;
+
+import java.net.URI;
+import java.time.Instant;
+import no.unit.nva.identifiers.SortableIdentifier;
+import no.unit.nva.publication.model.business.User;
+import no.unit.nva.publication.model.business.UserInstance;
+import no.unit.nva.publication.model.business.logentry.LogAgent;
+import no.unit.nva.publication.model.business.logentry.LogTopic;
+import no.unit.nva.publication.model.business.logentry.PublicationLogEntry;
+
+public record UpdatedResourceEvent(Instant date, User user, URI institution, SortableIdentifier identifier)
```

Comment:
It's a `date`, but it isn't; here as elsewhere

Reply:
Yeah, we discussed it before. There is a task for it :)
Changing it now will be a large refactor change + we need a migration for migrating this field in database.